src/Nri/Ui/Highlighter/V1.elm

```diff
@@ -613,19 +613,55 @@ groupContainer viewSegment highlightables =
                             [ Css.backgroundColor Css.transparent
                             , Css.Global.children
                                 [ Css.Global.selector ":first-child"
-                                    (MediaQuery.highContrastMode
-                                        [ Maybe.map
-                                            (\name ->
-                                                Css.before
-                                                    [ Css.property "content" ("\"[" ++ name ++ "] \"")
-                                                    ]
+                                    ((Maybe.map
```

Comment:
I wonder if this and the end-highlight Maybe.map logic could be simplified? I think since we always want a before and after pseudo-element, we could move the maybe.mapping "in" further, to where it's used:

```
(Css.property "content"
    ("\" [start"
        ++ (Maybe.map (\name -> " " ++ name)
                markedWith.name
                |> Maybe.withDefault ""
           )
        ++ "] \""
    )
    :: srOnlyStyles
)
```